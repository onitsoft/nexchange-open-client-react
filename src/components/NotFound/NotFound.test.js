import React from 'react';
import { mount, shallow, render } from 'enzyme';
import { NotFound } from './NotFound';

describe('NotFound', () => {
  it('renders correctly', () => {
    const goBack = jest.fn();
    const wrapper = render(<NotFound history={goBack} />);
    expect(wrapper).toMatchSnapshot();
  });
  // it('should call history.push', () => {
  //     const wrapper = mount(<MyComponent />);
  //     // perform your test
  // });
});
