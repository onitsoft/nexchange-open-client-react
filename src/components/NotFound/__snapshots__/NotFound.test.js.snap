// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
initialize {
  "0": Object {
    "attribs": Object {
      "class": "container",
      "id": "not-found",
    },
    "children": Array [
      Object {
        "attribs": Object {
          "class": "col-xs-12 col-sm-12 col-md-5 col-md-offset-1 col-lg-5 col-lg-offset-1",
        },
        "children": Array [
          Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "data": "error.notfound1",
                "next": null,
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
            ],
            "name": "h2",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {},
              "children": Array [
                Object {
                  "data": "error.notfound21 ",
                  "next": Object {
                    "attribs": Object {},
                    "children": Array [
                      Object {
                        "data": "error.notfound22",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "strong",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": Object {
                      "data": " error.notfound23",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "text",
                    },
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {},
                    "x-attribsPrefix": Object {},
                  },
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
                Object {
                  "attribs": Object {},
                  "children": Array [
                    Object {
                      "data": "error.notfound22",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "strong",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "data": " error.notfound23",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "text",
                  },
                  "parent": [Circular],
                  "prev": Object {
                    "data": "error.notfound21 ",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {},
                  "x-attribsPrefix": Object {},
                },
                Object {
                  "data": " error.notfound23",
                  "next": null,
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {},
                    "children": Array [
                      Object {
                        "data": "error.notfound22",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "strong",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": Object {
                      "data": "error.notfound21 ",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                    "type": "tag",
                    "x-attribsNamespace": Object {},
                    "x-attribsPrefix": Object {},
                  },
                  "type": "text",
                },
              ],
              "name": "h5",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": Object {
                "attribs": Object {},
                "children": Array [
                  Object {
                    "attribs": Object {
                      "class": "fas fa-long-arrow-alt-left",
                    },
                    "children": Array [],
                    "name": "i",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": Object {
                      "data": "error.notfound22",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "text",
                    },
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  Object {
                    "data": "error.notfound22",
                    "next": null,
                    "parent": [Circular],
                    "prev": Object {
                      "attribs": Object {
                        "class": "fas fa-long-arrow-alt-left",
                      },
                      "children": Array [],
                      "name": "i",
                      "namespace": "http://www.w3.org/1999/xhtml",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "tag",
                      "x-attribsNamespace": Object {
                        "class": undefined,
                      },
                      "x-attribsPrefix": Object {
                        "class": undefined,
                      },
                    },
                    "type": "text",
                  },
                ],
                "name": "a",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": null,
                "parent": [Circular],
                "prev": [Circular],
                "type": "tag",
                "x-attribsNamespace": Object {},
                "x-attribsPrefix": Object {},
              },
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {},
              "x-attribsPrefix": Object {},
            },
            "parent": [Circular],
            "prev": null,
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "data": "error.notfound21 ",
                "next": Object {
                  "attribs": Object {},
                  "children": Array [
                    Object {
                      "data": "error.notfound22",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "strong",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "data": " error.notfound23",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "text",
                  },
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "tag",
                  "x-attribsNamespace": Object {},
                  "x-attribsPrefix": Object {},
                },
                "parent": [Circular],
                "prev": null,
                "type": "text",
              },
              Object {
                "attribs": Object {},
                "children": Array [
                  Object {
                    "data": "error.notfound22",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "strong",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "data": " error.notfound23",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "text",
                },
                "parent": [Circular],
                "prev": Object {
                  "data": "error.notfound21 ",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
                "type": "tag",
                "x-attribsNamespace": Object {},
                "x-attribsPrefix": Object {},
              },
              Object {
                "data": " error.notfound23",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {},
                  "children": Array [
                    Object {
                      "data": "error.notfound22",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "strong",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": Object {
                    "data": "error.notfound21 ",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {},
                  "x-attribsPrefix": Object {},
                },
                "type": "text",
              },
            ],
            "name": "h5",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": Object {
              "attribs": Object {},
              "children": Array [
                Object {
                  "attribs": Object {
                    "class": "fas fa-long-arrow-alt-left",
                  },
                  "children": Array [],
                  "name": "i",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "data": "error.notfound22",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "text",
                  },
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                Object {
                  "data": "error.notfound22",
                  "next": null,
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {
                      "class": "fas fa-long-arrow-alt-left",
                    },
                    "children": Array [],
                    "name": "i",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "tag",
                    "x-attribsNamespace": Object {
                      "class": undefined,
                    },
                    "x-attribsPrefix": Object {
                      "class": undefined,
                    },
                  },
                  "type": "text",
                },
              ],
              "name": "a",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": null,
              "parent": [Circular],
              "prev": [Circular],
              "type": "tag",
              "x-attribsNamespace": Object {},
              "x-attribsPrefix": Object {},
            },
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {},
              "children": Array [
                Object {
                  "data": "error.notfound1",
                  "next": null,
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
              ],
              "name": "h2",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": null,
              "type": "tag",
              "x-attribsNamespace": Object {},
              "x-attribsPrefix": Object {},
            },
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
          Object {
            "attribs": Object {},
            "children": Array [
              Object {
                "attribs": Object {
                  "class": "fas fa-long-arrow-alt-left",
                },
                "children": Array [],
                "name": "i",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": Object {
                  "data": "error.notfound22",
                  "next": null,
                  "parent": [Circular],
                  "prev": [Circular],
                  "type": "text",
                },
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {
                  "class": undefined,
                },
                "x-attribsPrefix": Object {
                  "class": undefined,
                },
              },
              Object {
                "data": "error.notfound22",
                "next": null,
                "parent": [Circular],
                "prev": Object {
                  "attribs": Object {
                    "class": "fas fa-long-arrow-alt-left",
                  },
                  "children": Array [],
                  "name": "i",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": [Circular],
                  "parent": [Circular],
                  "prev": null,
                  "type": "tag",
                  "x-attribsNamespace": Object {
                    "class": undefined,
                  },
                  "x-attribsPrefix": Object {
                    "class": undefined,
                  },
                },
                "type": "text",
              },
            ],
            "name": "a",
            "namespace": "http://www.w3.org/1999/xhtml",
            "next": null,
            "parent": [Circular],
            "prev": Object {
              "attribs": Object {},
              "children": Array [
                Object {
                  "data": "error.notfound21 ",
                  "next": Object {
                    "attribs": Object {},
                    "children": Array [
                      Object {
                        "data": "error.notfound22",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "strong",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": Object {
                      "data": " error.notfound23",
                      "next": null,
                      "parent": [Circular],
                      "prev": [Circular],
                      "type": "text",
                    },
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "tag",
                    "x-attribsNamespace": Object {},
                    "x-attribsPrefix": Object {},
                  },
                  "parent": [Circular],
                  "prev": null,
                  "type": "text",
                },
                Object {
                  "attribs": Object {},
                  "children": Array [
                    Object {
                      "data": "error.notfound22",
                      "next": null,
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                  ],
                  "name": "strong",
                  "namespace": "http://www.w3.org/1999/xhtml",
                  "next": Object {
                    "data": " error.notfound23",
                    "next": null,
                    "parent": [Circular],
                    "prev": [Circular],
                    "type": "text",
                  },
                  "parent": [Circular],
                  "prev": Object {
                    "data": "error.notfound21 ",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                  "type": "tag",
                  "x-attribsNamespace": Object {},
                  "x-attribsPrefix": Object {},
                },
                Object {
                  "data": " error.notfound23",
                  "next": null,
                  "parent": [Circular],
                  "prev": Object {
                    "attribs": Object {},
                    "children": Array [
                      Object {
                        "data": "error.notfound22",
                        "next": null,
                        "parent": [Circular],
                        "prev": null,
                        "type": "text",
                      },
                    ],
                    "name": "strong",
                    "namespace": "http://www.w3.org/1999/xhtml",
                    "next": [Circular],
                    "parent": [Circular],
                    "prev": Object {
                      "data": "error.notfound21 ",
                      "next": [Circular],
                      "parent": [Circular],
                      "prev": null,
                      "type": "text",
                    },
                    "type": "tag",
                    "x-attribsNamespace": Object {},
                    "x-attribsPrefix": Object {},
                  },
                  "type": "text",
                },
              ],
              "name": "h5",
              "namespace": "http://www.w3.org/1999/xhtml",
              "next": [Circular],
              "parent": [Circular],
              "prev": Object {
                "attribs": Object {},
                "children": Array [
                  Object {
                    "data": "error.notfound1",
                    "next": null,
                    "parent": [Circular],
                    "prev": null,
                    "type": "text",
                  },
                ],
                "name": "h2",
                "namespace": "http://www.w3.org/1999/xhtml",
                "next": [Circular],
                "parent": [Circular],
                "prev": null,
                "type": "tag",
                "x-attribsNamespace": Object {},
                "x-attribsPrefix": Object {},
              },
              "type": "tag",
              "x-attribsNamespace": Object {},
              "x-attribsPrefix": Object {},
            },
            "type": "tag",
            "x-attribsNamespace": Object {},
            "x-attribsPrefix": Object {},
          },
        ],
        "name": "div",
        "namespace": "http://www.w3.org/1999/xhtml",
        "next": null,
        "parent": [Circular],
        "prev": null,
        "type": "tag",
        "x-attribsNamespace": Object {
          "class": undefined,
        },
        "x-attribsPrefix": Object {
          "class": undefined,
        },
      },
    ],
    "name": "div",
    "namespace": "http://www.w3.org/1999/xhtml",
    "next": null,
    "parent": null,
    "prev": null,
    "root": Object {
      "attribs": Object {},
      "children": Array [
        [Circular],
      ],
      "name": "root",
      "namespace": "http://www.w3.org/1999/xhtml",
      "next": null,
      "parent": null,
      "prev": null,
      "type": "root",
      "x-attribsNamespace": Object {},
      "x-attribsPrefix": Object {},
    },
    "type": "tag",
    "x-attribsNamespace": Object {
      "class": undefined,
      "id": undefined,
    },
    "x-attribsPrefix": Object {
      "class": undefined,
      "id": undefined,
    },
  },
  "_root": [Circular],
  "length": 1,
  "options": Object {
    "decodeEntities": true,
    "normalizeWhitespace": false,
    "withDomLvl1": true,
    "xml": false,
  },
}
`;
